README.md

```diff
@@ -4,6 +4,36 @@
 
 This repository contains the source for the Google-maintained Tomcat [docker](https://docker.com) image.
 
+# Building the image
```

Comment:
This type of information should probably go in a separate readme file called DEVELOPING.md, like we've done for the jetty and openjdk runtimes. You should reference the DEVELOPING.md file at the bottom of this one.